qiskit_nature/operators/second_quantization/fermionic_op.py

```diff
@@ -96,69 +128,55 @@ class FermionicOp(ParticleOp):
 
     """
 
+    _SPARSE_LABEL_PATTERN = re.compile(r"^([I\+\-NE]_\d\s)*[I\+\-NE]_\d(?!\s)$")
+    _DENSE_LABEL_PATTERN = re.compile(r"^[I\+\-NE]+$")
+
     def __init__(
-            self,
-            data: Union[str, Tuple[str, complex], List[Tuple[str, complex]]],
```

Comment:
I agreee with @woodsp-ibm and would like to keep the `Tuple` option available here :+1:

Reply:
OK